### PR TITLE
Fix firewall config installer

### PR DESCRIPTION
**Pull request by salvatore-coppola**
Solve #3418

### CODE REVIEW

File: kura/distrib/src/main/resources/intel-up2-centos-7/iptables.init [hunk 1]
```diff
@@ -1,71 +1,79 @@
 *nat
+:PREROUTING ACCEPT [0:0]
```
Comment (pierantoniomerlino) [inline kura/distrib/src/main/resources/intel-up2-centos-7/iptables.init]: Why this line? It is in line 5.

File: kura/distrib/src/main/resources/intel-up2-ubuntu-18/kura_install.sh [hunk 2]
```diff
@@ -88,6 +93,7 @@ mkdir -p /var/named
 chown -R bind /var/named
 cp ${INSTALL_DIR}/kura/install/named.ca /var/named/
 cp ${INSTALL_DIR}/kura/install/named.rfc1912.zones /etc/
+cp ${INSTALL_DIR}/kura/install/usr.sbin.named /etc/apparmor.d/
```
Comment (pierantoniomerlino) [inline kura/distrib/src/main/resources/intel-up2-ubuntu-18/kura_install.sh:96]: Named is used for configuring DNS servers. I really don't know if it is needed anymore or not...

File: kura/distrib/src/main/resources/intel-up2-centos-7/kura_install.sh [hunk 3]
```diff
@@ -52,15 +52,22 @@ rm /etc/resolv.conf
...
 chmod +x ${INSTALL_DIR}/kura/bin/.recoverDefaultConfig.sh
 
-cp ${INSTALL_DIR}/kura/install/iptables.init /etc/sysconfig/iptables
-cp /etc/sysconfig/iptables ${INSTALL_DIR}/kura/.data/iptables
+#set up default firewall configuration
+cp ${INSTALL_DIR}/kura/install/iptables.init ${INSTALL_DIR}/kura/.data/iptables
+chmod 644 ${INSTALL_DIR}/kura/.data/iptables
+cp ${INSTALL_DIR}/kura/.data/iptables /etc/sysconfig/iptables
+cp ${INSTALL_DIR}/kura/install/firewall.init ${INSTALL_DIR}/kura/bin/firewall
+chmod 755 ${INSTALL_DIR}/kura/bin/firewall
```
Comment (pierantoniomerlino) [inline kura/distrib/src/main/resources/intel-up2-centos-7/kura_install.sh:60]: Do we need also the firewall.service here?

File: kura/distrib/src/main/resources/intel-up2-centos-7/iptables.init (hunk 1 above)
Comment (salvatore-coppola) [reply kura/distrib/src/main/resources/intel-up2-centos-7/iptables.init]: removed

File: kura/distrib/src/main/resources/intel-up2-ubuntu-18/kura_install.sh (hunk 2 above)
Comment (salvatore-coppola) [reply kura/distrib/src/main/resources/intel-up2-ubuntu-18/kura_install.sh:96]: me neither...

File: kura/distrib/src/main/resources/intel-up2-centos-7/kura_install.sh (hunk 3 above)
Comment (salvatore-coppola) [reply kura/distrib/src/main/resources/intel-up2-centos-7/kura_install.sh:60]: does firewalld use iptables ?

Comment (pierantoniomerlino) [reply kura/distrib/src/main/resources/intel-up2-centos-7/kura_install.sh:60]: No, `firewalld` is a replacement of `iptables`. But the `firewall.service` is a service added by Kura that runs `/opt/eclipse/kura/bin/firewall`at startup.

Comment (salvatore-coppola) [reply kura/distrib/src/main/resources/intel-up2-centos-7/kura_install.sh:60]: i will add it

**Review by pierantoniomerlino — APPROVED** [review]

**Pull request merged**